loadTestUsers.py

```diff
@@ -126,6 +126,7 @@ def seed_cognito(test_users, user_pool_id, password):
         role = ""
         if "role" not in user:
             if "cms_roles" not in user:
+                print(f'no role or cms_roles for user:\n {user}')
```

Comment:
so, when I was reading up on "cherry-picking" commits, there was a warning about doing this, then merging more than once.  Should watch this... (or, maybe a merge from develop will remove it?)

Reply:
are you saying this is a line you later removed or otherwise altered in another commit?

Reply:
actually i see what you mean, worrying about duplicating the commit. I just merged in from dev and it recognized it as same commit and dropped it from the diff! woohoo!